config/routes.rb

```diff
@@ -4,4 +4,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :fishes, only: %i[index show new create] do
+    resources :bookings, only: %i[:new :create]
+  end
+  get "pages", to: "pages#about"
+  get "pages", to: "pages#my_bookings" do
```

Comment:
on va utiliser un controller "my_bookings" avec une action show

 get "my_bookings", to: "my_bookings#show"